deployment/README-django.md

```diff
@@ -35,34 +39,58 @@ $ ./scripts/cipublish
 ...
 ```
 
-It defaults to push to export ECHOLOCATOR_ENVIRONMENT="stgdjango".
+It defaults to push to export ECHOLOCATOR_ENVIRONMENT="stgdjango".  For production `export ECHOLOCATOR_ENVIRONMENT=prddjango` before cipublish.
 
 ## Terraform
 
+### New Environment
+
 First, we need to make sure there is a `terraform.tfvars` file in the project settings bucket on S3. The `.tfvars` file is where we can change specific attributes of the project's infrastructure, not defined in the `variables.tf` file.
 
 Here is an example `terraform.tfvars` for this project:
 
 ```hcl
 project     = "echo"
-environment = "stgdjango"
+environment = "prddjango"
 aws_region  = "us-east-1"
 
 r53_private_hosted_zone = "echo.internal"
+r53_public_hosted_zone  = "app.echosearch.org"
+
+rds_database_identifier = "echo-prddjango"
+rds_database_name       = "echo"
+rds_database_username   = "echo"
+rds_database_password   =
+
+fargate_app_cpu        = 1024
+fargate_app_memory     = 2048
+fargate_app_cli_cpu    = 256
+fargate_app_cli_memory = 1024
 
-rds_database_identifier = stgdjango
-rds_database_name       = echo
-rds_database_username   = echo
-rds_database_password   = echo
+rollbar_access_token =
+django_secret_key =
+
+default_from_email =
```

Comment:
We could probably fill `default_from_email` in since it is similar to the `r52_public_hosted_zone`: "noreply@app.echosearch.org".

Reply:
Added